src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -2989,6 +2989,20 @@ class DropTriggerStatementSegment(BaseSegment):
     )
 
 
+@postgres_dialect.segment()
+class InsertIntoAliasExpressionSegment(BaseSegment):
+    """A reference to an object with an `AS` clause.
+
+    AS is not optional in this segment.
+    """
+
+    type = "alias_expression"
+    match_grammar = Sequence(
+        "AS",
+        Ref("SingleIdentifierGrammar"),
+    )
```

Comment:
There is nothing in this to imply it's limited to `INSERT INTO` (except the segment name),and it seems fairly simple. Does it really need to be a separate segment?